### PR TITLE
Create eip-2228.md

### PR DESCRIPTION
**Pull request by fulldecent**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by MicahZoltu** [comment]
Seems like it should be `Ethereum Mainnet` as the canonical name, and `Mainnet` for short.  This comes up specifically in one of the examples where I think `Mainnet on Ethereum` is incorrect, it should read `Ethereum Mainnet` as `Ethereum` isn't a thing something can be "on", but `Ethereum Mainnet` or `Ethereum Ropsten` is something that something can be "on".

**Comment by MicahZoltu** [comment]
I dislike Informational EIPs, so I'll leave this EIP to someone else to do a full review of.

File: EIPS/eip-2228.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+author: William Entriken (@fulldecent)
+discussions-to: https://github.com/ethereum/EIPs/issues/2228
+status: Draft
+type: Informational
+created: 2019-08-04
+---
+
+## Simple Summary
+
+The Ethereum network with chainid 1 is referenced from authoritative sources using several names that are conflicting. This standard specifies the name for this network and provides reference examples in an effort to standardize the word choice and provide a common language for use to use to refer to this network.
```
Comment (MicahZoltu) [inline EIPS/eip-2228.md:13]: This summary is too long.  Thin email subject title or GitHub pull request title.  Perhaps something like:
> Defines the naming convention of Ethereum Mainnet.

Comment (MicahZoltu) [reply EIPS/eip-2228.md:13]: Also, this simple summary currently includes motivational content, which should be moved to the Motivation section.

**Review by lightclient — CHANGES_REQUESTED** [review]
One thing about Informational EIP is that many of the EIP headings no longer make sense. I recommend i) addressing Micah's feedback and ii) renaming the sections / reorganizing the EIP to flow more naturally - instead of being constrained via the EIP headings (e.g. backward compatibility, test cases, implementation, etc).

**Comment by fulldecent** [comment]
Summary is minified at 37e6e07f96009affd80e200d93902136e1433e24.

# This is an informational EIP and should not be merged [straw man argument]

Informational EIPs are in-scope for the EIPs project as defined at https://eips.ethereum.org/EIPS/eip-1 and this draft follows those rules.

# Sections should be renamed/excluded

I disagree. This Informational EIP draft is a specification for how to use language. Test cases and backwards compatibilities section are appropriate for specifications, and I have included them.

If the section heading names are obtuse this should be argued in the context of updating EIP-1 and it should be out of scope in this context of considering a technically-valid EIP draft for publication.

# Summary

I have addressed all technical concerns in this review and request please that this draft be published. Additional discussion on the draft can proceed as comments on the draft content.

File: EIPS/eip-2228.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+> Deploy your contract to the Ethereum main network.
+
+This is referring to a "main" network which is context-dependent. If you were reading this text on a page about Ethereum Classic, they would be referring to network_id: 2 / chain_id: 62. Therefore this word usage is less crisp. Do NOT use wording like this.
+
+> Connect to mainnet.
+
+These words literally mean nothing. The lowercase, not-proper-noun word "mainnet" is not a plain English word and it should not be in any dictionary. Do NOT use wording like this.
+
+## Implementation
```
Comment (lightclient) [inline EIPS/eip-2228.md:89]: I recommend removing this section.

Comment (lightclient) [inline EIPS/eip-2228.md:89]: I recommend removing this section.

File: EIPS/eip-2228.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+Anybody that has travelled to certain countries and seen an "IPhone [sic]" repair store should immediately recognize that this is off-brand and unofficial. Likewise, the astute customer of Ethereum should recognize if they see the name of the network referred to using different names in different software, so let's avoid this.
+
+## Backwards Compatibility
+
+- MetaMask currently uses the wording "Main Ethereum Network" in the account network chooser. In this context, it is understood that "main" is used as an adjective, not a proper noun form. Such usage is acceptable per this EIP. Also, MetaMask is using "Main" and "Network" with a capitalized first letter for stylistic purposes, not to designate a proper noun.
+
+- Certain references to "Mainnet" and alternate spellings are published in EIPs made in the following published EIPs 2, 7, 150, 155, 170 and 779.
+
+## Test Cases
```
Comment (lightclient) [inline EIPS/eip-2228.md:51]: I recommend renaming this to something more appropriate. "Test Cases" clearly refer to testing technical specifications, whereas this is a change in nomenclature. Something along the lines of "Usage" would be preferred.

**Review by lightclient — APPROVED** [review]
As this EIP meets the minimum requirements, I will accept it as a draft. Please consider the feedback regarding the section names. Thank you!

**Pull request merged**